src/js/hooks/cycleCount/useCountStep.jsx

```diff
@@ -55,7 +55,7 @@ const useCountStep = () => {
         currentLocation?.id,
         cycleCountIds,
       );
-      tableData.current = data?.data;
+      tableData.current = data?.data.filter((cycleCount) => cycleCount.countIndex !== 0);
```

Comment:
I think this should be `===`, I think on count step you want to see only count index 0 items.

Reply:
I tried: 
```
console.log(data?.data.filter((cycleCount) => cycleCount.countIndex !== 0));
```
and the output is:
![image](https://github.com/user-attachments/assets/a7e83a6a-e53b-46bc-a9a1-7fab16dd393f)

Reply:
@alannadolny it's because you are checking `cycleCount` status not the status of items for each cycle count.